cromwell_interact.py

```diff
@@ -360,6 +365,7 @@ def get_status(id, port,timeout=60, nocalls=False, minkeys=False,http_port=80):
     parser_submit.add_argument('--inputs', type=str, help='Path to wdl inputs')
     parser_submit.add_argument('--deps', type=str, help='Path to zipped dependencies file')
     parser_submit.add_argument('--label', type=str, help='Label of the workflow',default = '')
+    parser_submit.add_argument('--disable-monitoring',action="store_true",help='Disable task monitoring')
 
```

Comment:
could add optional argument --monitor pathtomonitor and default value is the hardcoded one and get rid of hardcoding

Reply:
so two options:
- --disable-monitoring for disabling monitoring
- --monitor VALUE for changing monitoring script path?

Reply:
yep